src/brasil/gov/portal/tests/test_views.py

```diff
@@ -463,3 +469,52 @@ def test_view_items(self):
         'link': False
     },
 ]
+
+
+class SummaryViewTestCase(BaseViewTestCase):
+
+    def login_browser(self):
+        """Autentica usuário de teste no browser"""
+        setRoles(self.portal, TEST_USER_ID, ['Site Administrator'])
+        self.browser.handleErrors = False
+        basic_auth = 'Basic {0}'.format(
+            '{0}:{1}'.format(TEST_USER_NAME, TEST_USER_PASSWORD)
+        )
+        self.browser.addHeader('Authorization', basic_auth)
+
+    def test_data_nao_pode_ser_1969_por_padrao_de_itens_criados(self):
+        with api.env.adopt_roles(['Manager', ]):
+            obj = api.content.create(
+                type='collective.nitf.content',
+                container=self.portal['folder'],
+                id='noticia',
+                title='noticia'
+            )
+            # Necessário para poder visualizar os objetos criados nos testes
+            # unitários em self.browser.
+            transaction.commit()
+            # Curioso que na template preciso fazer
+            # obj.EffectiveDate() mas não preciso obj.modified() e obj.effective()
+            # (só obj.modified já volta o valor) mas aqui no Python preciso disso.
+            item_date = obj.EffectiveDate() == 'None' and obj.modified() or obj.effective()
```

Comment:
isso ai deve ser escrito:

```python
item_date = obj.modified() if obj.EffectiveDate() == 'None' else obj.effective()
```

acho que também podem usar `obj.Date()` ao invés de isso acima.

Reply:
Pela lógica da template customizada, a idéia é mostrar a data de modificação e não a data de criação do objeto, e, como a linha que trata isso já havia sido aprovada num PR anterior, estamos mantendo a lógica que já estava na template mas adicionando um teste para evitar que esse erro ocorra novamente.

Reply:
de qualquer jeito o uso do `and/or` como operador ternário está ultrapassado e não é claro para pessoas que não conhecem esse truque.

Reply:
Verdade. Ainda mais depois da https://www.python.org/dev/peps/pep-0308/. Complementamos com a sua correção. Obrigado!